web/client/reducers/draw.js

```diff
@@ -49,7 +50,10 @@ function draw(state = initialState, action) {
             currentStyle: action.currentStyle
         });
     case GEOMETRY_CHANGED:
-        return assign({}, state, {tempFeatures: action.features});
+        let newData = action.features;
+        const normalizedData = newData[0].geometry.coordinates.map(i => i.map(x => x.map((item) => [normalizeLng(item[0]), item[1]])));
```

Comment:
when a geometry is changes you are not sure about the type of the geomtry you are changing. 
It can be a Point a LineString a Polygon or a MultiPolygon etc.

so you cannot take for granted that these nested mapping will work always and you have to add more checks before proceeding

Other than this units testing are missing and this is a perfect PR to add them. 
please add a unit test for each type of geometry
- Point
- LineString
- Polygon
- MultiPoint
- MultiLineString
- MultiPolygon

Reply:
Alright @MV88 , Let me add them

Reply:
Done writing unit tests @MV88